typer/completion.py

```diff
@@ -68,15 +68,13 @@ def _install_completion_placeholder_function(
     install_completion: bool = Option(
         None,
         "--install-completion",
-        is_flag=True,
         callback=install_callback,
         expose_value=False,
         help="Install completion for the current shell.",
     ),
     show_completion: bool = Option(
         None,
         "--show-completion",
-        is_flag=True,
```

Comment:
I'm not sure why we had these `is_flag` specifications here. I think we can just remove them, because we removed the `parameter_info.is_flag is not False` condition.